tests/test_outdated.py

```diff
@@ -28,6 +28,11 @@ class Package:
     version_spec: str | None
     version_conda_forge: str | None = None
 
+
+allowed_outdated: set[str] = {
```

Comment:
Why a set though and not just a list?

Reply:
Uniqueness isn't a bad idea?

Reply:
It was really meant as an open question :) thanks for the explanation, makes sense to me!

Reply:
And that's just my guess :) Maybe there is a different reason.